app/views/products/_form.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= form_for @product do |form| %>
+  <%= form.text_field :name %>
+  <%= form.text_field :price %>
+  <%= form.text_field :short_description %>
+
+  <%= form.submit t(".submit_product") %>
+<% end %>
```

Comment:
A lot of our apps use simple_form .. not that you need to change it but wanted to bring it up as something to be aware of and maybe brush up on using.